.travis.yml

```diff
@@ -1,7 +1,19 @@
 language: scala
 jdk: oraclejdk8
 scala: 2.12.4
-script: sbt test
+
+jobs:
+  include:
+    - stage: test
+      script: sbt test
+    - stage: publish
+      script: sbt publish
+
+stages:
+  - name: test # runs on master commits and PRs
+    if: NOT tag =~ ^v
+  - name: publish # runs on main repo master commits or version-tagged commits
+    if: repo = lightbend/sbt-paradox-apidoc AND ( ( branch = master AND type = push ) OR tag =~ ^v )
```

Comment:
So snapshots as well? Do we have a cleanup mechanism for those yet?

Reply:
No cleanup strategy yet. For smaller projects it probably is not needed, as there are few snapshots.